docs/meeting-minutes/TrenchBoot-AEM-January24-2023.md

```diff
@@ -0,0 +1,62 @@
+## Attendees
+
+- Daniel P. Smith (Apertus Solutions)
+- Andrew Cooper (Citrix)
+- Krystian Hebel (3mdeb)
+- Michał Żygowski (3mdeb)
+
+## Agenda
+
+- Review the current state of TrenchBoot boot protocol being upstreamed to GRUB
+  and Linux kernel
+- Discuss the scope of the work needed to align Qubes OS Anti Evil Maid
+  solution based on TrenchBoot with the most recent boot protocol version
+- Discuss possibilities to implement DRTM launch on S3 resume in Xen
+
+## Meeting Minutes
+
+Current approach of booting Linux with TrenchBoot:
+
+- meet Linux demand so that anything can do EBS and DRTM can still work
+- Linux doing EBS to gather necessary info from firmware
+- bootloader to put slaunch code to memory
+- the code being put to memory is going to be standardized and does everything
+  what DRTM technology needs
+- standardize an input table to be used for generic stub not specific to any
+  bootloader/kernel
+
+Current Xen status:
+
+- Xen efi needs fixing in order to work without Xen's efi-stub, so it would be
+  better to follow Linux's approach of jumping into Xen and Xen calling out to
+  slstub
+- AP wakeup developed during the Qubes OS AEM PoC in Xen hypervisor kernel is
+  still valid and will be needed in the most recent approach as well
+- for legacy boot module will have to be placed in memory by GRUB but GRUB will
+  also construct the input table and call SENTER/SKINIT
```

Comment:
Upstream, I'm only aware of support for AP booting after SKINIT.  And even then, "won't crash if booted like this", and not really "PoC of working trenchboot" quality.  I don't recall any patches for the Intel SENTER support.

Reply:
@andyhhp true, there were no patches yet. We still are putting things together and just a few days ago, we finished the whole integration and verification. What I meant here, is that the work we did for AP bring up after SENTER will be needed for both UEFI and legacy boot mode. The patch set shall be sent when new get everything rebase onto most recent boot protocol. Does that approach sounds good to you @andyhhp ?